src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -0,0 +1,83 @@
+import { html, TemplateResult } from "lit-html";
+import { dapps } from "./dapps";
+
+/** A teaser for the dapps explorer, acting as a button */
+export const dappsTeaser = ({
+  click,
+}: {
+  click: () => void;
+}): TemplateResult => {
+  return html`<article class="c-card c-card--narrow">
+    <h3 class="t-title t-title--discrete" role="presentation">
+      Dapps explorer
+    </h3>
+    <h2 class="t-title">Explore dapps</h2>
+    <button
+      class="c-click-area"
+      @click="${() => click()}"
+      aria-label="Show list of dapps"
+    >
+      <figure class="c-card__teaser c-marquee">${marqueeList()}</figure>
```

Comment:
Does `c-card__teaser` have an effect here? I couldn't see any difference when removing it

Reply:
It did not before, but does now, because i merged in the master that has the `--bezel-x` and `--bezel-y` vars set.